backend/.env.example_docker

```diff
@@ -1,10 +1,9 @@
 PORT=8000
-MONGO_URI=mongo/fightpandemics
```

Comment:
Not sure if this file is meant to be edited? @robinv85

Reply:
@snowiesuet Good catch! 

@alexjmcm This file should NOT be edited. The purpose of this file was to set an outline on how to create a `backend/.env` file. 

I also noticed that you deleted `backend/.env.example_local` and `client/.env.example`. There was no need to delete them. Please properly follow the setup in the README on how to add environment variables.